cmd/gke-exec-auth-plugin/main.go

```diff
@@ -57,6 +61,15 @@ func main() {
 	var token string
 	var err error
 
+	// Lock the process, this prevents parallel gke-exec-auth-plugin
+	// invocations from making redundant CSR requests.
+	fileLock := flock.New(*flockPath)
+	err = fileLock.Lock()
+	if err != nil {
```

Comment:
Only lock when you need to refresh the kubelet cert.

Reply:
Done.